docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -0,0 +1,752 @@
+pod/sample-domain1-managed-server1   1/1     Running   0          40m
+pod/sample-domain1-managed-server2   1/1     Running   0          40m
+
+NAME                                       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+service/sample-domain1-admin-server        ClusterIP   None           <none>        7001/TCP   41m
+service/sample-domain1-cluster-cluster-1   ClusterIP   100.66.99.27   <none>        8001/TCP   40m
+service/sample-domain1-managed-server1     ClusterIP   None           <none>        8001/TCP   40m
+service/sample-domain1-managed-server2     ClusterIP   None           <none>        8001/TCP   40m
+
+
+```
+
+#### Invoke the web application
+
+Create a load balancer to access the WebLogic Server Administration Console and applications deployed in the cluster.
+Tanzu supports the MetalLB load balancer and NGINX ingress for routing.
+
+
+Install the MetalLB load balancer by running following commands:
+
+```bash
+## create namespace metallb-system
+$ kubectl create ns metallb-system
+
+## deploy MetalLB load balancer
+$ kubectl apply -f https://raw.githubusercontent.com/google/metallb/v0.9.2/manifests/metallb.yaml -n metallb-system
+
+## create secret
+$ kubectl create secret generic -n metallb-system memberlist --from-literal=secretkey="$(openssl rand -base64 128)"
+
+$ cat metallb-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: default
+      protocol: layer2
+      addresses:
+      - 192.168.100.50-192.168.100.65
+
+$ kubectl apply -f metallb-configmap.yaml
+configmap/config created
+
+$ kubectl get all -n metallb-system
+NAME                              READY   STATUS    RESTARTS   AGE
+pod/controller-684f5d9b49-jkzfk   1/1     Running   0          2m14s
+pod/speaker-b457r                 1/1     Running   0          2m14s
+pod/speaker-bzmmj                 1/1     Running   0          2m14s
+pod/speaker-gphh5                 1/1     Running   0          2m14s
+pod/speaker-lktgc                 1/1     Running   0          2m14s
+
+NAME                     DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                 AGE
+daemonset.apps/speaker   4         4         4       4            4           beta.kubernetes.io/os=linux   2m14s
+
+NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/controller   1/1     1            1           2m14s
+
+NAME                                    DESIRED   CURRENT   READY   AGE
+replicaset.apps/controller-684f5d9b49   1         1         1       2m14s
+```
+
+
+Install NGINX.
+
+```bash
+$ helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+$ helm repo update
+$ helm install ingress-nginx ingress-nginx/ingress-nginx
+```
+
+
+Create ingress routing for accessing the application deployed in the cluster.
+
+```bash
+$ cat cluster-ingress.yaml
```

Comment:
Why two ingresses one for the cluster and one for admin console?  Can you not do 2 paths?  This is an example with traefik
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: traefik-pathrouting-1
  namespace: sample-domain1-ns
  annotations:
    kubernetes.io/ingress.class: traefik
spec:
  rules:
  - host:
    http:
      paths:
      - path: /
        backend:
          serviceName: sample-domain1-cluster-cluster-1
          servicePort: 8001
      - path: /console
        backend:
          serviceName: sample-domain1-admin-server
          servicePort: 7001

Reply:
Good point, merged both and used single yaml to create both access paths.